### ISSUE TITLE
First message in client_server example never receives a response from server 

### ISSUE THREAD

**Issue by joseluis** [amethyst/laminar#275]
Hi, I want to know if this is a bug or expected behavior.

The first message from the client to the server en the client_sever example never gets a response. Only the second message and the next ones. 

E.g. starting the server and then the client:
```
➜ cargo run --example server_client
    Finished dev [unoptimized + debuginfo] target(s) in 1.16s
     Running `target/debug/examples/server_client`
Please type in `server` or `client`.
s
Starting server..
Received "msg1" from V4(127.0.0.1)
Received "msg2" from V4(127.0.0.1)
Client timed out: 127.0.0.1:12352
```

```
$ cargo run --example server_client 
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
     Running `target/debug/examples/server_client`
Please type in `server` or `client`.
c
Starting client..
Connected on 127.0.0.1:12352
Type a message and press Enter to send. Send `Bye!` to quit.
msg1
Silence..
msg2
Server sent: Copy that!
```

**Comment by TimonPost** [comment]
Hi, do you have any sample code?

**Comment by DuckerMan** [comment]
@TimonPost There's code in #273

**Comment by joseluis** [comment]
@TimonPost Do you mean a different code from the [`server_client` example](https://github.com/amethyst/laminar/blob/master/examples/server_client.rs)?.

**Comment by joseluis** [comment]
This is fixed now, probably in #280
